src/feedspora/generic_client.py

```diff
@@ -240,6 +242,7 @@ def set_common_opts(self, account):
         if 'url_shortener' in account:
             self._url_shortener = account['url_shortener'].lower()
 
+        self._url_shortener_opts = dict()
```

Comment:
Same here: the `_url_shortener_opts = dict()` should be in an `else`  statement.

Reply:
OK, but we need to work out the test functionality question below first, I think.